module1/homework/calculate/calculate.hpp

```diff
@@ -3,5 +3,19 @@
 
 std::string calculate(const std::string& command, int first, int second) {
     // TODO: Implement your solution here and return proper value
-    return "";
+    if (command == "add"){
+        return std::to_string(first + second);
+    } else
+     if (command == "subtract"){
+        return std::to_string(first - second);
+    } 
+     if (command == "multiply"){
+        return std::to_string(first * second);
+    }  if (command == "divide"){
+        if(second == 0){
+             return "Division by 0";;
```

Comment:
you have two semicolons here
```suggestion
             return "Division by 0";
```